### PR TITLE
Ria 3861 ooc refuse

### PR DESCRIPTION
**Pull request by Lee365**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-3861

### Change description ###
RIA-3861 - LR submits a OOC appeal - Protection claim

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: e2e/flows/start-appeal.flow.ts [hunk 1]
```diff
@@ -37,13 +37,17 @@ export class StartAppealFlow {
...
 
-    async completeHomeOfficeReference(clickContinue = false, homeOfficeReferenceNumber = '') {
+    async completeHomeOfficeReference(clickContinue = false, ooc = false, homeOfficeReferenceNumber = '') {
         if (homeOfficeReferenceNumber !== '') {
             await this.ccdFormPage.setFieldValue('Home Office Reference/Case ID', homeOfficeReferenceNumber);
         } else {
             await this.ccdFormPage.setFieldValue('Home Office Reference/Case ID', '01234567');
         }
-        await this.ccdFormPage.setFieldValue('Enter the date the decision letter was sent', '{$TODAY}');
+        if (ooc) {
```
Comment (divisathyan) [inline e2e/flows/start-appeal.flow.ts:46]: If it is ooc, why are we setting date sent to Today?

**Review by divisathyan — CHANGES_REQUESTED** [review]
Please see my comment

File: e2e/flows/start-appeal.flow.ts (hunk 1 above)
Comment (divisathyan) [reply e2e/flows/start-appeal.flow.ts:46]: Also, how is feature enabled flag used?

Comment (Lee365) [reply e2e/flows/start-appeal.flow.ts:46]: ooc date sent set to Today-10

Comment (Lee365) [reply e2e/flows/start-appeal.flow.ts:46]: This feature is only for ooc

**Review by Srihari-Nakkala — APPROVED** [review]
LGTM

**Review by mkayad — DISMISSED** [review]
LGTM

**Review by jkumar-moj — APPROVED** [review]

**Pull request merged**